src/lib/ngsi/AttributeDomainName.cpp

```diff
@@ -99,11 +99,11 @@ void AttributeDomainName::present(const std::string& indent)
 {
   if (string != "")
   {
-    PRINTF("%sAttributeDomainName: %s\n", indent.c_str(), string.c_str());
+    LM_F(("%sAttributeDomainName: %s", indent.c_str(), string.c_str()));
```

Comment:
So I understand that PRINTF will not be used any more, in favour of LM_F. Is that correct? In that case, we would need to to an overall replacement of PRINTF for LM_F. We should open a tech debt issue nice2have for this.